.pre-commit-config.yaml

```diff
@@ -4,6 +4,15 @@ repos:
     hooks:
       - id: pyupgrade
         args: [--py39-plus]
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v1.7.7
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --remove-duplicate-keys
+          - --ignore-init-module-imports
```

Comment:
Why would we ignore this?

Reply:
It's the safer default, but sure enough, there was just one noncompliant re-exporting `__init__` file, fixed that up in the next commit.